### ISSUE TITLE
Right command does not fires on macOS

### ISSUE THREAD

**Issue by KirillSuhodolov** [jaywcjlove/hotkeys#92]
```
hotkeys('ctrl+r, command+r', function() {
  alert('stopped reload!');
  return false;
});
```

from example

left command fires, but right not, on macOS

**Comment by jaywcjlove** [comment]
@KirillSuhodolov https://codepen.io/jaywcjlove/pen/xorRvW

```js
hotkeys('ctrl+r, command+r', function(event,handler) {
  event.preventDefault();
  console.log('handle', handler.key,  event)
});
```

**Comment by bichotll** [comment]
I can confirm that. (Right) command+r does not work for me either.

**Comment by KirillSuhodolov** [comment]
example not working too

**Comment by jaywcjlove** [comment]
@KirillSuhodolov I tested no problem.
![image](https://user-images.githubusercontent.com/1680273/61171527-3ba26580-a5ab-11e9-85cb-ecd7a96c996b.png)

**Comment by VikasShashidhar** [comment]
This does not work for me either - https://jsfiddle.net/zrtaLgop/

**Comment by VikasShashidhar** [comment]
@jaywcjlove Any updates on this please?

**Comment by jaywcjlove** [comment]
![image](https://user-images.githubusercontent.com/1680273/62004909-0296f300-b15e-11e9-8601-d68980295787.png)

```diff
- hotkeys("option+k,cmd+k", function(event, handler) {
+ hotkeys("option+k,command+k", function(event, handler) {
  console.log('handler:', handler.shortcut);
});
```

@VikasShashidhar I tested no problem, I don't know if it is a browser version issue.

**Comment by VikasShashidhar** [comment]
@jaywcjlove It's weird! I am able to make this work on Firefox but not on Chrome or Safari. What version of Chrome are you trying this on?

Also, it's only the right `cmd` that doesn't work. The right `option` works just fine.

![Right Command Bug](https://user-images.githubusercontent.com/27778544/62110328-cd101800-b2cb-11e9-9310-a468e57b4121.gif)

**Comment by jaywcjlove** [comment]
@VikasShashidhar Google Chrome v75.0.3770.142

**Comment by VikasShashidhar** [comment]
@jaywcjlove I am using the exact same version and running into this issue. I am not sure how else I can help you repro this. Any suggestions?

**Comment by jaywcjlove** [comment]
@VikasShashidhar Can I run your example?

**Comment by VikasShashidhar** [comment]
Of course. Please go ahead. You can use the fiddle that I shared to do that.

**Comment by felixreichle** [comment]
Had the same issue, moved line 193-195 in main.js to line 187. This fixed it for me, not sure if it's breaking something else though.

**Comment by jaywcjlove** [comment]
@felixreichle There is no problem with the code moving up.

https://github.com/jaywcjlove/hotkeys/blob/be2c9b716473a731cb8a9091e8180c27ec0677f3/src/main.js#L186-L196

**Comment by jaywcjlove** [comment]
@KirillSuhodolov @VikasShashidhar @felixreichle 
I don't know if I can solve the problem.
